pkg/apis/keda/v1alpha1/zz_generated.openapi.go

```diff
@@ -11,17 +11,17 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"./pkg/apis/keda/v1alpha1.AuthEnvironment":           schema_pkg_apis_keda_v1alpha1_AuthEnvironment(ref),
-		"./pkg/apis/keda/v1alpha1.AuthPodIdentity":           schema_pkg_apis_keda_v1alpha1_AuthPodIdentity(ref),
-		"./pkg/apis/keda/v1alpha1.AuthSecretTargetRef":       schema_pkg_apis_keda_v1alpha1_AuthSecretTargetRef(ref),
-		"./pkg/apis/keda/v1alpha1.ObjectReference":           schema_pkg_apis_keda_v1alpha1_ObjectReference(ref),
-		"./pkg/apis/keda/v1alpha1.ScaleTriggers":             schema_pkg_apis_keda_v1alpha1_ScaleTriggers(ref),
-		"./pkg/apis/keda/v1alpha1.ScaledObject":              schema_pkg_apis_keda_v1alpha1_ScaledObject(ref),
-		"./pkg/apis/keda/v1alpha1.ScaledObjectAuthRef":       schema_pkg_apis_keda_v1alpha1_ScaledObjectAuthRef(ref),
-		"./pkg/apis/keda/v1alpha1.ScaledObjectSpec":          schema_pkg_apis_keda_v1alpha1_ScaledObjectSpec(ref),
-		"./pkg/apis/keda/v1alpha1.ScaledObjectStatus":        schema_pkg_apis_keda_v1alpha1_ScaledObjectStatus(ref),
-		"./pkg/apis/keda/v1alpha1.TriggerAuthentication":     schema_pkg_apis_keda_v1alpha1_TriggerAuthentication(ref),
-		"./pkg/apis/keda/v1alpha1.TriggerAuthenticationSpec": schema_pkg_apis_keda_v1alpha1_TriggerAuthenticationSpec(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.AuthEnvironment":           schema_pkg_apis_keda_v1alpha1_AuthEnvironment(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.AuthPodIdentity":           schema_pkg_apis_keda_v1alpha1_AuthPodIdentity(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.AuthSecretTargetRef":       schema_pkg_apis_keda_v1alpha1_AuthSecretTargetRef(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.ObjectReference":           schema_pkg_apis_keda_v1alpha1_ObjectReference(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.ScaleTriggers":             schema_pkg_apis_keda_v1alpha1_ScaleTriggers(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.ScaledObject":              schema_pkg_apis_keda_v1alpha1_ScaledObject(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.ScaledObjectAuthRef":       schema_pkg_apis_keda_v1alpha1_ScaledObjectAuthRef(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.ScaledObjectSpec":          schema_pkg_apis_keda_v1alpha1_ScaledObjectSpec(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.ScaledObjectStatus":        schema_pkg_apis_keda_v1alpha1_ScaledObjectStatus(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.TriggerAuthentication":     schema_pkg_apis_keda_v1alpha1_TriggerAuthentication(ref),
+		"github.com/kedacore/keda/pkg/apis/keda/v1alpha1.TriggerAuthenticationSpec": schema_pkg_apis_keda_v1alpha1_TriggerAuthenticationSpec(ref),
```

Comment:
Do you know what's the deal with these changes? when I build your branch on my machine, it reverts them back to `"./pkg/..."` Is it because I moved the repo out of GOPATH?

Reply:
It might be it, but I am not really sure honestly

Reply:
but that's generated stuff, so we should not tuch thouse files directly